### PR TITLE
Updates Data Prepper use-case images

### PR DESCRIPTION
**Pull request by dlvenable**
### Description

This PR updates the images used for Data Prepper use-cases:

* Updated icon for Data Prepper
* Use Open Sans font
* Makes use of OpenSearch style guide for colors
* Use JPEG files for compression

### Issues Resolved
N/A

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Naarcha-AWS — APPROVED** [review]

**Review by cwillum — APPROVED** [review]
LGTM

File: _data-prepper/common-use-cases/trace-analytics.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ When using Data Prepper as a server-side component to collect trace data, you ca
 
 The following flowchart illustrates the trace analytics workflow, from running OpenTelemetry Collector to using OpenSearch Dashboards for visualization.
 
-<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-components.png" alt="Trace analyticis component overview">{: .img-fluid}
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-components.jpg" alt="Trace analyticis component overview">{: .img-fluid}
```
Comment (hdhalter) [inline _data-prepper/common-use-cases/trace-analytics.md:18]: alt="Trace analyticis --> alt="Trace analytics"

File: _data-prepper/common-use-cases/trace-analytics.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ To monitor trace analytics, you need to set up the following components in your
 
 To monitor trace analytics in Data Prepper, we provide three pipelines: `entry-pipeline`, `raw-trace-pipeline`, and `service-map-pipeline`. The following image provides an overview of how the pipelines work together to monitor trace analytics.
```
Comment (hdhalter) [inline _data-prepper/common-use-cases/trace-analytics.md:28]: Data Prepper provides three pipelines that can be used to monitor trace analytics:

File: _data-prepper/common-use-cases/log-analytics.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ Data Prepper is an extendable, configurable, and scalable solution for log inges
 
 The following image shows all of the components used for log analytics with Fluent Bit, Data Prepper, and OpenSearch.
 
-![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.png)
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.jpg)
 
 In the application environment, run Fluent Bit. Fluent Bit can be containerized through Kubernetes, Docker, or Amazon Elastic Container Service (Amazon ECS). You can also run Fluent Bit as an agent on Amazon Elastic Compute Cloud (Amazon EC2). Configure the [Fluent Bit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper. Then deploy Data Prepper as an intermediate component and configure it to send the enriched log data to your OpenSearch cluster. From there, use OpenSearch Dashboards to perform more intensive visualization and analysis.
```
Comment (hdhalter) [inline _data-prepper/common-use-cases/log-analytics.md:16]: I know I'm not obligated to review this, but I just wonder if we can start this sentence with some kind of intro to what they are doing? Like, "To get started using log analytics in Data Prepper, run Fluent Bit in the application environment of your choice. ...  Also, the sentence with "Then..." should have a better starter phrase, like, "After configuring Fluent Bit, deploy Data Prepper..." (The paragraph seems a bit rushed.)

File: _data-prepper/common-use-cases/log-analytics.md [hunk 4]
```diff
@@ -11,15 +11,15 @@ Data Prepper is an extendable, configurable, and scalable solution for log inges
 
 The following image shows all of the components used for log analytics with Fluent Bit, Data Prepper, and OpenSearch.
 
-![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.png)
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.jpg)
 
 In the application environment, run Fluent Bit. Fluent Bit can be containerized through Kubernetes, Docker, or Amazon Elastic Container Service (Amazon ECS). You can also run Fluent Bit as an agent on Amazon Elastic Compute Cloud (Amazon EC2). Configure the [Fluent Bit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper. Then deploy Data Prepper as an intermediate component and configure it to send the enriched log data to your OpenSearch cluster. From there, use OpenSearch Dashboards to perform more intensive visualization and analysis. 
 
 ## Log analytics pipeline
```
Comment (hdhalter) [inline _data-prepper/common-use-cases/log-analytics.md:18]: Should it be Log analytics pipelines?

File: _data-prepper/common-use-cases/trace-analytics.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ When using Data Prepper as a server-side component to collect trace data, you ca
 
 The following flowchart illustrates the trace analytics workflow, from running OpenTelemetry Collector to using OpenSearch Dashboards for visualization.
 
-<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-components.png" alt="Trace analyticis component overview">{: .img-fluid}
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-components.jpg" alt="Trace analyticis component overview">{: .img-fluid}
 
 To monitor trace analytics, you need to set up the following components in your service environment:
```
Comment (hdhalter) [inline _data-prepper/common-use-cases/trace-analytics.md:20]: To monitor trace analytics, you need to set up the --> To monitor trace analytics, set up

**Review by hdhalter — COMMENTED** [review]
Other than a few nits, looks good!

**Review by kolchfa-aws — COMMENTED** [review]
LGTM

**Pull request merged**